airgun/entities/base.py

```diff
@@ -6,3 +9,32 @@ def __init__(self, browser):
         self.browser = browser
         self.session = browser.extra_objects['session']
         self.navigate_to = self.session.navigator.navigate
+
+    def create_bookmark(self, values, search_query=None):
+        """Create a bookmark.
+
+        :param dict values: dictionary with keys 'name', 'query', 'public'
+        :param str optional search_query: if something different from `query`
+            should be typed into searchbox
+        """
+        # not using separate navigator step here not to have to register
+        # navigate step for every single entity
+        view = self.navigate_to(self, 'All')
+        if not search_query:
+            search_query = values.get('query')
+        if not hasattr(view, 'search'):
+            raise KeyError(
+                '{} does not have searchbox'.format(self.__class__.__name__))
+        view.search(search_query)
```

Comment:
Why do we need to do search? and not just input value?

Reply:
Thought we need to perform search so query can be automatically filled. Turns out we don't need to do that, just typing into search field is enough. Will update, thanks for suggestion.